_drafts/SC1-2-1+SC1-2-3-video-transcript.md

```diff
@@ -0,0 +1,113 @@
+---
+name: video element transcript
+group: SC1-2-video-element
+description: |
+  Non-streaming video elements must have all audio and visual information available in a transcript.
+
+success_criterion:
+- 1.2.1 # Audio-only and Video-only (Prerecorded)
+- 1.2.3 # Audio Description or Media Alternative (Prerecorded)
+- 1.2.8 # Media Alternative (Prerecorded)
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+- Audio output
+- Visual output
+
+authors:
+- Wilco Fiers
+- Brian Bors
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any [non-streaming][] video element [visible in the page][]
+
+### Expectation 1
+
+A [text transcript][] is available for the test target, either on the page or available through a link (or some other mechanism).
```

Comment:
Expand on `some other mechanism`, by providing a few example?

Reply:
A text transcript (or anything for that matter) is always available on either this page, or on another page (available trough a link) so we have decided to simply drop "(or some other mechanism)".